csi/server/examples/kubernetes/nginx.yaml

```diff
@@ -41,4 +46,4 @@ spec:
   - name: csi-data-opensdsplugin
     persistentVolumeClaim:
       claimName: csi-pvc-opensdsplugin
-      readOnly: false
+      readOnly: false
```

Comment:
Please add a blank line to remove the error here.

Reply:
Done.